R/RadeonProRender/build_tarballs.jl

```diff
@@ -7,7 +7,7 @@ version = v"2.2.9"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderSDK.git", "5916c58f9cc037899e0aa85e5f5a67c6e1ee7b29")
+    GitSource("https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderSDK.git", "e284dfac04ee3a142603ba955a2bf8d11a78d945")
```

Comment:
You probably need to change the version number as well? :slightly_smiling_face:

Reply:
Yes, thanks :D - did not notice that at first.  I also changed the Julia compat as you suggested in https://github.com/JuliaPackaging/Yggdrasil/pull/4001#issuecomment-984564572.

Reply:
Cool, so that wasn't impossible after all :sweat_smile: